.gitignore

```diff
@@ -33,4 +33,5 @@ coverage.txt
 # Emacs save files have literal pound signs/octothorpes/hashtags in them
 # So the \# escapes # as comment syntax
 \#*#
-.\#*
\ No newline at end of file
+.\#*
+sagalog/*
```

Comment:
Great Suggestion @dgassaway I created a .scootdata folder which can contain misc garbage data files that should not be checked in.  I updated the config to put fileSagaLog files here by default.  I also added deleting the files in this directory to make clean.  This directory is ignored.  

@JeanetteBruno at the end of make travis I invoke clean-data, this way we don't pollute those machines.  I'm hesitant to just have recoverytest blow it away incase people were relying on data that is in there for a previous run.